### PR TITLE
RSDK-2583 - Update vision-service-examples with mlmodel service

### PR DESCRIPTION
**Pull request by kharijarrett**

### CODE REVIEW

File: go/cmd.go [hunk 1]
```diff
@@ -2,45 +2,46 @@ package main
...
+)
+
+const (
+	pathHere = "/full/path/to/vision-service-examples/go/"
+	saveTo   = "output.jpg"
 )
 
 func main() {
+	// Setup robot client
 	logger := golog.NewDevelopmentLogger("opencv")
```
Comment (bhaney) [inline go/cmd.go]: change the name of the logger since we don't use go-cv anymore

**Review by bhaney — APPROVED** [review]
Nice! Just one string edit

**Review by bhaney — CHANGES_REQUESTED** [review]
Just noticed, you have to edit the README.md to reflect the changes in the vision service as well

File: go/cmd.go (hunk 1 above)
Comment (kharijarrett) [reply go/cmd.go]: Very valid. Done.

**Review by bhaney — APPROVED** [review]
LGTM!

**Pull request merged**